src/main/cms/cms_menu_blackbox.c

```diff
@@ -188,6 +189,23 @@ static const void *cmsx_EraseFlash(displayPort_t *pDisplay, const void *ptr)
 
     return MENU_CHAIN_BACK;
 }
+
+static const void *cmsx_StorageDevice(displayPort_t *pDisplay, const void *ptr)
+{
+    UNUSED(ptr);
+
+    if (!flashfsIsSupported()) {
+        return NULL;
+    }
+
+    displayClearScreen(pDisplay, DISPLAY_CLEAR_WAIT);
+    displayWrite(pDisplay, 5, 3, DISPLAYPORT_SEVERITY_INFO, "STORAGE DEVICE MODE");
+    displayRedraw(pDisplay);
+    beeper(BEEPER_USB);
+    systemResetToMsc(0);
+
+    return MENU_CHAIN_BACK;
```

Comment:
(we should mark `systemResetToMsc` as noreturn.)

Reply:
return NULL after systemResetToMsc
otherwise return MENU_CHAIN_BACK

Reply:
Ok. Done.

Reply:
Can remove 221